src/generator/AutoRest.Ruby.Azure.Tests/RspecTests/paging_spec.rb

```diff
@@ -129,6 +129,21 @@
     expect(items.count).to eq(10)
   end
 
+  it 'should get multiple pages with fragmented next link' do
+    page = @client.paging.get_multiple_pages_fragment_next_link_as_lazy('1.6', 'test_user')
+    expect(page.is_a? PagingModule::Models::ProductResult)
+    expect(page.odatanext_link).not_to be_nil
+
+    all_item = page.get_all_items
+    expect(all_item.count).to eq(10)
+
+    # Get all items at once

```

Comment:
>  # Get all items at once [](start = 2, length = 25)

feel free to split this into a separate test if you'd like, I've done that with others.